.github/workflows/npm-publish.yml

```
@@ -38,7 +38,7 @@ jobs:
           script: |
             process.chdir("./packages/codemirror-lang-dentaku");
             const packageJson = require("./package.json");
-            const script = require('.github/scripts/bump-version');
+            const script = require('../../.github/scripts/bump-version');
```

I think this also works if you don't want it to be relative to the current file.
```suggestion
            const script = require('./.github/scripts/bump-version');
```